### ISSUE TITLE
Rcon in language C#

### ISSUE THREAD

**Issue by ScorpionsSM** [csprance/MisRCON#41]
Hello, I would like to know, is it possible to write in the language C#? And in which direction to look, what is the principle of work on the server

**Comment by csprance** [comment]
check out [node-misrcon](https://github.com/Csprance/node-misrcon) for more info on how the rcon protocol works.

**Comment by ScorpionsSM** [comment]
I didn’t really understand anything, I didn’t learn that language, I can’t understand where is md5 and xml

**Comment by Spafbi** [comment]
Yes, it's a different language, but learn just enough to figure out how the
protocol works. Then apply that new knowledge to implement the protocol in
whatever language you wish.

**Comment by ScorpionsSM** [comment]
You offer me to learn the language in order to understand the protocol? ...

**Comment by Spafbi** [comment]
Yes, as that's how this open source stuff works. I did it for my personal
RCON library - looked at code in other languages (I pulled bits from RCON
libraries written in at least three different languages) and adapted it to
my preferred programming language. You might be able to find someone to
create a C# implementation for you, but neither cSprance nor myself have
the time or inclination to do so - we tend to stick to our languages of
choice; his preferred language is TypeScript, mine is Python.
